src/app/services/sms/sms.util.ts

```diff
@@ -18,3 +32,57 @@ export const renderVerificationSms = (
 ): string => dedent`Use the OTP ${otp} to submit on ${appHost}.
 
   Never share your OTP with anyone else. If you did not request this OTP, you can safely ignore this SMS.`
+
+/**
```

Comment:
if the retrieval handler is in AdminFormController, then we can reuse admin-form.utils too

Reply:
sure, will shift this over to `admin-form.utils`